Analytics/Helpers/SEGBluetooth.m

```diff
@@ -28,7 +28,10 @@ - (id)init {
 
   _queue = dispatch_queue_create("io.segment.bluetooth.queue", NULL);
 
-  if (&CBCentralManagerOptionShowPowerAlertKey != NO) {
+  // Reading the address inline fails on iOS 6, must use a temporary
+  // variable to compare.
+  NSInteger addr = &CBCentralManagerOptionShowPowerAlertKey;
+  if (addr != 0) {
```

Comment:
Ah! I see what you mean. No, we should not do what you have specified above. The check is to see if the symbol `CBCentralManagerOptionShowPowerAlertKey` exists, so checking for its address is the correct thing to do (the symbol does not exist on iOS6). I could do 

    BOOL show = &CBCentralManagerOptionShowPowerAlertKey;
    if (show != NO) {
    ...

but IMHO that obfuscates the meaning of getting the address. Does that make sense?